api/graphite.go

```diff
@@ -741,19 +761,23 @@ func (s *Server) graphiteTagFindSeries(ctx *middleware.Context, request models.G
 		return
 	}
 
-	response.Write(ctx, response.NewJson(200, series, ""))
+	seriesNames := make([]string, 0, len(series))
+	for _, serie := range series {
+		seriesNames = append(seriesNames, serie.Pattern)
+	}
+	response.Write(ctx, response.NewJson(200, seriesNames, ""))
 }
 
-func (s *Server) clusterFindByTag(ctx context.Context, orgId int, expressions []string, from int64) ([]string, error) {
-	seriesSet := make(map[string]struct{})
+func (s *Server) clusterFindByTag(ctx context.Context, orgId int, expressions []string, from int64) ([]Series, error) {
```

Comment:
This needed to be changed so that we could get all the metric defs from teh remote nodes to help choose consolidations and aggregations that are available.